tools/run_tests/run_tests.py

```diff
@@ -622,13 +622,13 @@ class PythonConfig(
 class PythonLanguage(object):
 
     _TEST_SPECS_FILE = {
-        'native': 'src/python/grpcio_tests/tests/tests.json',
-        'gevent': 'src/python/grpcio_tests/tests/tests.json',
-        'asyncio': 'src/python/grpcio_tests/tests_aio/tests.json',
+        'native': ['src/python/grpcio_tests/tests/tests.json'],
+        'gevent': ['src/python/grpcio_tests/tests/tests.json', 'src/python/grpcio_tests/tests_gevent/tests.json',],
```

Comment:
Line 784: Let's disable Python27 gevent tests for Linux and macOS. You can do so by not returning 27's Python config.